content/tracing/visualization/resource.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Resource page
+kind: Documentation
+further_reading:
+- link: "/tracing/setup/"
+  tag: "Documentation"
+  text: Learn how to setup APM tracing with your application
+- link: "/tracing/visualization/services_list/"
+  tag: "Documentation"
+  text: Discover the list of services reporting to Datadog
+- link: "/tracing/visualization/service"
+  tag: "Documentation"
+  text: Learn more about services in Datadog
+- link: "/tracing/visualization/trace"
+  tag: "Documentation"
+  text: Understand how to read a Datadog Trace
+---
+
+{{< img src="tracing/visualization/resource/ressource.png" alt="Ressource" responsive="true" popup="true">}}
+
+See the list of resources associated with your service. Resources are particular actions for your services (typically individual endpoints or queries). Read more about resources in [Getting Started with APM](/tracing/visualization). Sort the resources for this service by requests, latency, errors, and time, to identify areas of high traffic or potential trouble. Note that the these metric columns are configurable (see image below).
+
+## Out of the box graphs
+
+Datadog provides out of the box graphs for any given Resource:
+
+* Requests - Choose to display:
+    *  The **Total amount of requests** 
+    *  The amount of **Requests per second**
+* Latency -  Choose to display:
+    *  The Avg/p75/p90/p95/p99/Max latency of your traced requests 
+* Error - Choose to display:
+    * The **Total amount of errors** 
+    * The amount of **Errors per second** 
+    * The **% Error Rate** 
+* Sub-Services: When there is multiple service involved a 4th graph is available that breaks down your **Total time spent**/**%of time spent**/**Avg time per request** of your service by *services* or *type*. For services like *Postgres* or *Redis*, which are "final" operations not calling other service underneath, there will be no Sub-services graph.
+
+{{< img src="tracing/visualization/resource/resource_otb_graphs.png" alt="Out of the bow resource graphs" responsive="true" popup="true" style="width:90%;">}}
+
+**Note**: Use the *cogs* icon to display all options available for any given graph.
+
+### Export to Timeboard
+
+On the upper-right corner of each graphs click on the little arrow in order to export your graph into a pre-existing [Timeboard](/graphing/dashboards/timeboard):
+
+{{< img src="tracing/visualization/resource/save_to_timeboard.png" alt="Save to timeboard" responsive="true" popup="true" style="width:40%;">}}
+
+### Latency distribution
+
+In addition to all those graph there is a resource latency distribution graph
```

Comment:
Should there be a period or colon after this sentence?

Reply:
👌